src/main/java/com/keboola/tableexporter/Application.java

```diff
@@ -104,14 +109,27 @@ private static void connectDb() throws ApplicationException, UserException {
             try {
                 connectionString.append("jdbc:oracle:thin:@").append(tnsnamesService);
                 System.out.println("Connecting user " + dbUser + ". Using service name " + tnsnamesService + " from tnsnames.ora.");
-                connection = DriverManager.getConnection(connectionString.toString(), connectionProps);
+                connection = (OracleConnection) DriverManager.getConnection(connectionString.toString(), connectionProps);
             } catch (SQLException ex) {
                 throw new UserException("Connection error: " + ex.getMessage(), ex);
             }
         }
+
+        // Proxy user
+        if (dbProxyUser != null) {
+            java.util.Properties proxyProp = new java.util.Properties();
+            proxyProp.put(OracleConnection.PROXY_USER_NAME, dbProxyUser);
+            System.out.println("Proxy user = \"" + dbProxyUser + "\"");
+
+            try {
+                connection.openProxySession(OracleConnection.PROXYTYPE_USER_NAME, proxyProp);

```

Comment:
Podla dokumentacie, by sa to malo pridat takto.

Reply:
https://docs.oracle.com/database/121/JJDBC/proxya.htm#JJDBC28352